spec/spec_helper.rb

```diff
@@ -27,7 +36,7 @@ def temporary_table_exists?(table_name)
     end
   end
 
-  class SQLite3Adapter
+  class SQLite3Adapter < AbstractAdapter
```

Comment:
Mhm, this sounds related to the [previous PR](https://github.com/jpignata/temping/pull/20). 
Just out of curiosity, was it failing without this?